config/initializers/seeds.rb

```diff
@@ -0,0 +1,51 @@
+return unless (DatabaseSeed.table_exists? rescue(false)) && !Foreman.in_rake?('db:rollback')
+
+Foreman::Application.configure do |app|
```

Comment:
This feels like we're duplicating the logic from seed.rb? perhaps the hash checking should move there and the initializer should just trigger the rake task? or the logic extracted to a service that both call?

Reply:
Absolutely, I should have mentioned that as a to-do still. My initial goal was get a feel for the implementation which sounds like folks are good with.